server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -1366,6 +1366,14 @@ public enum Config {
             "200",
             "The default maximum primary storage space (in GiB) that can be used for an account",
             null),
+DefaultMaxAccountProjects(
```

Comment:
adding the config here is a bit of a deprecated way of doing these. Have a look at ConfigKey<> for the prefered way.
Define a ConfigKey<> in a manager implementation or a service interface and then add the configKey to the return of the `getConfirKeys()` in the manager. You can then use the value more dynamically and it doesn't have to be preloaded in the config method.

I understand you are just following the convention you see in the files you are changing but still,...